src/main/java/com/powsybl/network/hypothesis/HypothesisUtils.java

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.network.hypothesis;
+
+import com.powsybl.iidm.network.*;
+
+import java.util.function.BiConsumer;
+
+/**
+ * Some useful utility methods to create network hypotheses.
+ *
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public final class HypothesisUtils {
+
+    /**
+     * Split a given line and create a fictitious voltage level at the junction.<br>
+     * The line is considered split in half such as this:<br>
+     * <code>r1 = 0.5 * r</code><br>
+     * <code>r2 = 0.5 * r</code><br>
+     * If the line is between two BUS-BREAKER voltage levels, the fictitious voltage level will be BUS-BREAKER and the two new lines
+     * will be linked to a fictitious bus in this voltage level.<br>
+     * If the line is between the two NODE-BREAKER voltage levels, the fictitious voltage level will be NODE_BREAKER and the
+     * two lines will be linked via an internal connection between two nodes in this voltage level.
```

Comment:
Should we also support the mixed topology lines? If yes, then we should also add a TopologyKind argument, to choose which topology will be the new voltage level